pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -171,10 +198,20 @@ func NewElasticsearchKubeControllers(cfg *KubeControllersConfiguration) *kubeCon
 		kubeControllerAllowTigeraPolicy = esKubeControllersAllowTigeraPolicy(cfg)
 	}
 
-	enabledControllers := []string{"authorization", "elasticsearchconfiguration"}
+	var enabledControllers []string
+
+	if !cfg.Tenant.MultiTenant() {
+		enabledControllers = append(enabledControllers, "authorization", "elasticsearchconfiguration")
+		if cfg.ManagementCluster != nil {
+			enabledControllers = append(enabledControllers, "managedcluster")
+		}
+	} else {
+		enabledControllers = append(enabledControllers, "managedclusterlicensing")
+	}
 
-	if cfg.ManagementCluster != nil {
-		enabledControllers = append(enabledControllers, "managedcluster")
+	kubeControllerConfigName := "elasticsearch"
+	if cfg.Tenant.MultiTenant() {
+		kubeControllerConfigName = fmt.Sprintf("tenant-%s", cfg.Tenant.Spec.ID)
```

Comment:
I think this code should be reverted - we're not creating one per tenant any more IIUC

Reply:
Yup. I have this removed, forgot to push.